src/core/wb.js

```diff
@@ -57,9 +57,33 @@ var getUrlParts = function( url ) {
 
 	/**
 	 * @variable i18n
-	 * @return {string} of HTML document language
+	 * @return {string} of WET document language
 	 */
-	lang = document.documentElement.lang,
+	lang = ( function( ele ) {
+		let lang = document.documentElement.lang;
+		const shortLangLength = 2;
+
+		// Perform extra checks if the page uses a long language code
+		if ( lang.length > shortLangLength ) {
+			let longLangs = [ "pt-BR", "zh-Hans" ]; // Built-in long language codes
```

Comment:
Regarding the request in last week's technical review meeting to move this variable's declaration near here: https://github.com/wet-boew/wet-boew/blob/12d829e8a12940be2c0c89e80898e47e151efb1f/src/core/wb.js#L145

I don't think that'd be feasible. The ``lang`` function runs before ``wb`` gets declared. So if I were to move ``longLangs`` down there, it would be unavailable at the moment it's needed the most. Or maybe I'm missing something?

Reply:
How the default setting is set in the core is not consistent. So it is fine to have it here.